theano/compile/function_module.py

```diff
@@ -752,9 +753,33 @@ def checkSV(sv_ori, sv_rpl):
         return f_cpy
 
     def __call__(self, *args, **kwargs):
+        """
+        Evaluates value of a function on given arguments.
+
+        Parameters
+        ----------
+        args : list
+            List of inputs to the function. All inputs are required, even when
+            some of them are not necessary to calculate requested subset of
+            outputs.
+
+        kwargs : dict
+            TODO: other kwargs?
+            Keyword argument `output_subset` is a list of indices of the
+            function's outputs that are requested to be calculated.
```

Comment:
A previous version was using this interface:

a_theano_function.apply(*inputs, output_subset) or something similar. The alternatives I see to fix the issue of conflict name:

1) disallow named variable with the name output_subste
2) use a different way to call Theano function, like a_theano_function.apply(...)
3) Remove calling with named inputs. (I don't like that much, but I'm not strongly again this).